From: Markus Koschany <apo@debian.org>
Date: Sun, 8 Dec 2013 18:05:52 +0100
Subject: glShaderSource

Fix glshaderSource stub signature to match that in <GL/glext.h>.
Thanks to Colin Watson.

Forwarded: http://springrts.com/mantis/view.php?id=4208
---
 rts/lib/headlessStubs/glstub.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rts/lib/headlessStubs/glstub.c b/rts/lib/headlessStubs/glstub.c
index 0d41f15..1ade8a3 100644
--- a/rts/lib/headlessStubs/glstub.c
+++ b/rts/lib/headlessStubs/glstub.c
@@ -130,7 +130,7 @@ GLAPI void APIENTRY glGetShaderSource(GLuint shader, GLsizei bufSize, GLsizei *l
 GLAPI void APIENTRY glCompileShader(GLuint shader) {}
 GLAPI void APIENTRY glGetShaderiv(GLuint shader, GLenum pname, GLint *params) {}
 GLAPI void APIENTRY glGetShaderInfoLog(GLuint shader, GLsizei bufSize, GLsizei *length, GLchar *infoLog) {}
-GLAPI void APIENTRY glShaderSource (GLuint shader, GLsizei count, const GLchar* *string, const GLint *length) {}
+GLAPI void APIENTRY glShaderSource (GLuint shader, GLsizei count, const GLchar *const*string, const GLint *length) {}
 
 GLAPI void APIENTRY glUniform1fARB(GLint location, GLfloat v0) {}
 GLAPI void APIENTRY glUniform2fARB(GLint location, GLfloat v0, GLfloat v1) {}
